client/src/App.js

```diff
@@ -1,5 +1,161 @@
+  }
+
+  // Filter Function für Salads
+  function filterSalads(month) {
+    const filterSalads = salads.filter((salad) => salad.months.includes(month));
+    setSaladsToRender(filterSalads);
+  }
+
+  // Filter Function für Fruties
+  function filterFruities(month) {
+    const filterFruities = fruities.filter((fruity) =>
+      fruity.months.includes(month)
+    );
+    setFruitiesToRender(filterFruities);
+
+    // Filter Function für Month
+    const filterMonth = months.find((mon) => mon.number === month);
+    setMonth(filterMonth.name);
+  }
+
+  // Function um alles anzeigen zu lassen
+  function resetAll() {
+    setVeggiesToRender(veggies);
+    setFruitiesToRender(fruities);
+    setSaladsToRender(salads);
+    setMonth("Gesamtes Jahr");
+  }
+
+  function filterAll(month) {
+    filterVeggies(month);
+    filterFruities(month);
+    filterSalads(month);
+  }
+
+  return (
+    <div>
+      <main>
+        <h2>{month}</h2>
+        <WrapButton>
+          <Button onClick={() => filterAll(1)}>Jan</Button>
+          <Button onClick={() => filterAll(2)}>Feb</Button>
+          <Button onClick={() => filterAll(3)}>Mär</Button>
+          <Button onClick={() => filterAll(4)}>Apr</Button>
+          <Button onClick={() => filterAll(5)}>Mai</Button>
+          <Button onClick={() => filterAll(6)}>Jun</Button>
+          <Button onClick={() => filterAll(7)}>Jul</Button>
+          <Button onClick={() => filterAll(8)}>Aug</Button>
+          <Button onClick={() => filterAll(9)}>Sep</Button>
+          <Button onClick={() => filterAll(10)}>Okt</Button>
+          <Button onClick={() => filterAll(11)}>Nov</Button>
+          <Button onClick={() => filterAll(12)}>Dez</Button>
+          <Button onClick={resetAll}>Alle</Button>
+        </WrapButton>
+        <h3>Obst</h3>
+        <WrapContainer>
+          {frutiesToRender.map((fruty, index) => (
+            <PictureCard
+              key={index}
+              imageUrl={fruty.imageUrl}
+              name={fruty.name}
+            />
+          ))}
+        </WrapContainer>
+        <h3>Gemüse</h3>
+        <WrapContainer>
+          {veggiesToRender.map((veggie, index) => (
+            <PictureCard
+              key={index}
+              imageUrl={veggie.imageUrl}
+              name={veggie.name}
+            />
+          ))}
+        </WrapContainer>
+        <h3>Salat</h3>
+        <WrapContainer>
+          {saladsToRender.map((salads, index) => (
+            <PictureCard
+              key={index}
+              imageUrl={salads.imageUrl}
+              name={salads.name}
+            />
+          ))}
+        </WrapContainer>
+      </main>
+    </div>
+  );
 }
 
 export default App;
+
+const WrapContainer = styled.section`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const WrapButton = styled.section`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const Button = styled.button`
+  xdisplay: flex;
+  xflex-direction: row;
+  xalign-items: center;
```

Comment:
Remove the styles that you no longer use before you merge it into main :) You still have it in your git history if you want to bring it back :)